htdocs/wp-content/themes/owc-formulieren/assets/js/editor/personal-heading/save.js

```diff
@@ -0,0 +1,57 @@
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+
+const Save = ( { attributes } ) => {
+
+	return (
+		<div className="openpdd-personal-heading">
+			<div className="openpdd-personal-heading__content">
+				<h1 className="openpdd-personal-heading__header">
+					<span className="dynamic openpdd-personal-heading__morning">
+						{__( 'Goedemorgen', 'owc-formulieren' )}
+					</span>
+					<span className="dynamic openpdd-personal-heading__afternoon">
+						{__( 'Goedemiddag', 'owc-formulieren' )}
+					</span>
+					<span className="dynamic openpdd-personal-heading__evening">
+						{__( 'Goedenavond', 'owc-formulieren' )}
+					</span>
+					<span>
+						Brigitta!
```

Comment:
Voor nu prima, maar dit moet een refractor worden. 

Een collega is in een plugin bezig met (micro) blokjes die deze gegevens ophaalt. Dat kan dan hier d.m.v.  `<InnerBlocks.content/> `